### PR TITLE
bpo-40196: Fix a bug in the symtable when reporting inspecting global variables

### PR DESCRIPTION
**Pull request by pablogsal**
https://bugs.python.org/issue40196

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Fix a bug in the ``symtable`` module that was causing incorrectly report
```
Comment (corona10) [inline Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst]: nit
```
:mod:`symtable`
```

Comment (pablogsal) [reply Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst]: Oh, I was under the impression that both produce the same sphinx output?

Comment (corona10) [reply Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst]: FYI, it generates different output, The latter creates a hyperlink to the module
```
``symtable``
```
스크린샷 2020-04-07 오전 12 04 52

```
:mod:`symtable`
```

스크린샷 2020-04-07 오전 12 04 10

Comment (corona10) [reply Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst]: So end-user can move to the module more easily :)

Comment (pablogsal) [reply Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst]: Wow, I have been living in a lie! Thanks for pointing this out :)

Comment (pablogsal) [inline Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst]:
```suggestion
Fix a bug in the :mod:`symtable` module that was causing incorrectly report
```

**Review by isidentical — APPROVED** [review]

**Pull request merged**